src/components/EventCrud/EventCrud.jsx

```diff
@@ -1,30 +1,33 @@
 import { useState, useEffect } from 'react';
-import EventsList from '../EventList/EventList';
 import useEvents from '../../hooks/use-events'
 import MentorList from '../MentorList/MentorList'
 import MentorEventsList from '../MentorToEventsList/MentorToEventsList'
+import useMentors from '../../hooks/use-mentors';
+import useMentorEvents from '../../hooks/use-mentor-events';
 import './EventCrud.css'
 
 function EventCrud({activeEvent, onCreateEventClick, createEventOpen}) {
 
     const { events,  isEventsLoading, isEventsError } = useEvents();
+    const { mentors, isMentorsLoading, isMentorsError } = useMentors();
+    const { mentorevents, isMentorEventsLoading, isMentorEventsError } = useMentorEvents()
 
+    // Selected mentors to add
     const [mentorsToAdd, setMentorsToAdd] = useState([])
-
     const onMentorsAdd = (mentorAddList) => {
         setMentorsToAdd(mentorAddList)
     }
 
+    // Mentors currently linked to event that can be removed
     const [mentorsToRemove, setMentorsToRemove] = useState([])
-
     const onRemoveMentors = (mentorRemoveList) => {
         setMentorsToRemove(mentorRemoveList)
     }
 
-    const [currentMentors, setCurrentMentors] = useState([])
-
-    const onCurrentMentorsChange = (currentMentorsList) => {
-        setCurrentMentors(currentMentorsList)
+    
+    //TODO should be utils
```

Comment:
move this to utilities - there's a lot going on in these files and if it's being reused that's a win.

Reply:
Agree! Will migrate to your utils one at some stage. The only bit that will need the current way is in Edit Event Form, where the date-picker wants dash-delimited date (in order to display previous date that user is editing)